styles/section.css

```diff
@@ -0,0 +1,17 @@
+/* body {
```

Comment:
Change the 'section.css' name to be more descriptive of what it's styling

Reply:
Great suggestion, I'll modify the name to make it more clear!